### PR TITLE
Fix #2378

### LINKED ISSUE #2378: Report spam possible with PM/E-mail report medium
You can send countless reports about the same user/post/reputation, the duplicate checks only work if ModCP reporting is enabled (proably because the report is saved to DB only then, haven't looked yet). I'm not sure why. It can be an annoyance if someone notices which medium is used and tries to abuse it.

### PR DESCRIPTION
**Pull request by dequeues**

### CODE REVIEW

**Comment by Destroy666x** [comment]
https://github.com/mybb/mybb/issues/2378

The global report bar seems to be displayed, it should be disabled depending on the setting (and anything similar as well).

**Comment by dequeues** [comment]
Thought I added that, my bad.

File: global.php [hunk 1]
```diff
@@ -627,7 +627,7 @@
 		$can_access_moderationqueue = false;
 	}
 
-	if($can_access_moderationqueue || ($mybb->user['ismoderator'] && $mybb->usergroup['canmodcp'] == 1 && $mybb->usergroup['canmanagereportedcontent'] == 1))
+	if($can_access_moderationqueue || ($mybb->user['ismoderator'] && $mybb->usergroup['canmodcp'] == 1 && $mybb->usergroup['canmanagereportedcontent'] == 1) && $mybb->settings['reportmethod'] == "db")
```
Comment (Destroy666x) [inline global.php]: It'd be more optimal at the beginning of https://github.com/dequeues/mybb/blob/fix2378/global.php#L592

**Comment by Destroy666x** [comment]
I see you changed the order but not the line - https://github.com/dequeues/mybb/blob/fix2378/global.php#L592 The whole block is for report bar.

**Comment by Destroy666x** [comment]
The bar conditional is still not fully correct if you test it (doesn't work) - parentheses are missing, `&&` doesn't work as supposed to.

File: admin/modules/home/index.php [hunk 2]
```diff
@@ -260,13 +260,16 @@
 	$query = $db->simple_select("posts", "COUNT(*) AS newposts", "dateline > '$timecut' AND visible='1'");
 	$newposts = my_number_format($db->fetch_field($query, "newposts"));
 
-	// Get the number of reported post
-	$query = $db->simple_select("reportedcontent", "COUNT(*) AS reported_posts", "type = 'post' OR type = ''");
-	$reported_posts = my_number_format($db->fetch_field($query, "reported_posts"));
+	// If report medium is MCP...
```
Comment (Destroy666x) [inline admin/modules/home/index.php]: No `if` here either. Besides, only the unread number is redundant due to no "mark as read" option. The other one may still be useful to see how many reports were sent.

**Comment by dequeues** [comment]
I honestly have no idea why the if statements got removed.

**Pull request merged**